src/engine/platform/sound/nes_nsfplay/nes_fds.cpp

```diff
@@ -9,7 +9,7 @@ const int RC_BITS = 12;
 
 NES_FDS::NES_FDS ()
 {
-    option[OPT_CUTOFF] = 2000;
+    option[OPT_CUTOFF] = 99999;

```

Comment:
Sorry, not related to pull request.........

Reply:
FDS cutoff was put back at 99999 to decrease bit resolution distortion (for now)